events/Harad.txt

```diff
@@ -166,6 +168,13 @@ country_event = {
 		}
 		ai_chance = {
 			factor = 75
+			modifier = {
+				factor = 100
+				OR = {
+					THIS = { is_our_vassal = FROM }
+					THIS = { in_sphere = FROM }
+				}
+			}
```

Comment:
If memory serves, having FROM in an ai_chance will cause a crash. Have you tested this, and confirmed in the logs that the ai got this decision without causing a crash?

Reply:
It doesn't, I actually used it in the Harondor colonization proposals.

Reply:
I'm a little suspicious, but good to know -- we start seeing crashes, we should look at this first and see if updating the FROM to a hardcoded tag fixes it